spec/function.dd

```diff
@@ -1026,18 +1026,21 @@ class Bar : Foo
 ------
 )
 
-        $(P Virtual functions all have a hidden parameter called the
-        $(I this) reference, which refers to the class object instance for which
-        the function is called.
+        $(P 
+            Non-static (i.e., virtual functions and `final` member
+            functions) all have a hidden parameter called the
+            `this` reference, which refers to the class object for which
+            the function is called.
         )
 
         $(P
             Functions with `Objective-C` linkage have an additional hidden,
             unnamed, parameter which is the selector it was called with.
         )
 
-        $(P To directly call a base member function, insert
-        base class name before the member function name. For example:
+        $(P To directly call a base member function (i.e., avoid dynamic
+        binding on member function call), insert the base class name before
+        the member function name. For example:
```

Comment:
AFAIU, the principle reason for this is to call a function in a non-overridable way, of which avoiding a virtual function call is a nice side-effect.

Reply:
Counterpoint, the purpose of avoiding dynamic binding on a member function call is to call a function in a non-overrideable way, so it is semantically and logically equivalent IMO

Reply:
Fair enough.